.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,37 @@
+settings:
+    # Jira project key to create the issue in
+  jira_project_key: "DOCPR"
+  
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done 
+    
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  # components:
+  #  - IoT
+  # - DACH TT
+      
+  # (Optional) GitHub labels. Only issues with one of those labels will be synchronized.
+  # If not specified, all issues will be synchronized
+  #labels:
+  #  - bug
+  #  - custom
+      
+  # (Optional) (Default: false) Add a new comment in GitHub with a link to Jira created issue
+  add_gh_comment: true
```

Comment:
Not sure whether this makes sense since our Jira issues aren't public, but the GitHub issues are.

Reply:
Agreed, will revert this to false

Reply:
After discussions, going to retain this to true since it is useful to have the Jira link in GitHub. For users who don't have access, it wouldn't open anyway.